dataeng/jobs/analytics/AnswerDistribution.groovy

```diff
@@ -28,7 +28,10 @@ class AnswerDistribution {
                     if (env_config.get('SNITCH')) {
                         shell('curl https://nosnch.in/' + env_config.get('SNITCH'))
                     }
-                }
+                    if (env_config.get('OPSGENIE_HEARTBEAT_NAME') && env_config.get('OPSGENIE_HEARTBEAT_KEY')){
```

Comment:
I'm not sure that the HEARTBEAT_KEY will show up in env_config -- I expect it in allVars.